### DISCUSSION TITLE
Can this tool also be used to convert csv / xls to reqif ?

### DISCUSSION THREAD

**Discussion by vesper8** [strictdoc-project/reqif#171] [Q&A]
With your help I was able to get your tool to convert REQIF to json. But I was wondering if your tool supports the opposite, exporting json or csv or xls to the REQIF format ?

I can't seem to find any information about this so I'm assuming it doesn't support it but wanted to double check with you.

Thanks!

**Comment by stanislaw** [comment]
Hi! There is no complete implementation but only an early draft of how it could work here: https://github.com/strictdoc-project/reqif/tree/main/tests/integration/examples/10_excel_convert_excel_to_reqif. 

Most recently I didn't find enough time to complete this implementation.
